CenterForOpenScience/osf.io
# [ENG-1268] add enable_chronos waffle flag

## Purpose

Add an `enable_chronos` waffle flag to be used to show the Chronos submission widget in the front end.

## Changes

* add flag to osf/features.py
* add migration to add the flag to the DB

## QA Notes

  - Does this change require a data migration?
*no*
  - What is the level of risk?
*low*
    - Any permissions code touched?
*no*
    - Is this an additive or subtractive change, other?
*additive*
  - How can QA verify?
*API* or *AdminAdminApp*?
    - If verifying through API, what's the new version?
*n/a*
  - What features or workflows might this change impact?
*Chronos submission*
  - How will this impact performance?
*n/a*

## Documentation

n/a

## Side Effects

n/a

## Ticket

[ENG-1268]

[ENG-1268]: https://openscience.atlassian.net/browse/ENG-1268
